rules-tests/Orm30/Rector/MethodCall/SetParametersArrayToCollectionRector/Fixtures/array-with-parameters.php.inc

```diff
@@ -0,0 +1,27 @@
+<?php
+
```

Comment:
like other PR, please add namespace if possible

Reply:
i will rebase this PR and apply all the fixes after the other one is done if this is fine for you